gtkm/auth/service_auth.py

```diff
@@ -1,13 +1,80 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Depends, HTTPException, Response, status
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import RedirectResponse, JSONResponse
+from sqlalchemy.orm import Session
+import requests
+import uuid
 
+from .database.database import Base, engine, get_db
+from .database.user_schema import User
+from .database.database_operations import get_user, get_user_by_github_login, create_user
+from .sessions.session import backend, cookie, verifier
+from .sessions.session_data import SessionData
+
+
+Base.metadata.create_all(bind=engine)
 auth = APIRouter()
 
+CLIENT_ID = "<CLIENT_ID>"
+CLIENT_SECRET = "<CLIENT_SECRET>"
```

Comment:
Jeszcze tylko ostatnia rzecz.   
Proponowałbym, aby te zmienne były wyciągane z enva

` os.environ.get('CLIENT_ID'),`

Reply:
Ja to zrobisz to już można mergować

Reply:
Done